Material.Blazor/Components/FloatingActionButton/MBFloatingActionButton.razor.cs

```diff
@@ -92,7 +92,8 @@ protected override async Task OnInitializedAsync()
 
             ConditionalCssClasses
                 .AddIf("mdc-fab--mini mdc-fab--touch", () => Type == MBFloatingActionButtonType.Mini)
```

Comment:
is `mdc-fab-touch` never required for non-mini FABs?

Reply:
Only for mini. The rule Google follow is that you need a touch target if a control is less than 48px x 48px for accessibility reasons.

https://github.com/material-components/material-components-web/tree/master/packages/mdc-fab#making-fabs-accessible